site/templates/nav/user_logged_out.jet

```diff
@@ -1,5 +1,6 @@
 {{block nav_user_logged_out()}}
   <s72-user-anon class="user-anon">
+    <div class="vl"></div>
```

Comment:
What's this about?

Reply:
It's a vertical line that seperates the login/logout from the search and language selector. It is purely visual. I would use a border for this but in the designs, it is shorter than the height of the container. As far as I'm aware, borders can only be the full height.

Reply:
Ah. I was confused because I thought it was `v1` like version one, but it's `vl` for vertical line. Perhaps spell it out in full.
```suggestion
    <div class="vertical-line"></div>
```

Remember that we can push back on anything for discussion. In this case you could easily justify making it a border rather than having another HTML element purely for visuals.